shell/assets/translations/en-us.yaml

```
@@ -1313,6 +1313,8 @@ cluster:
         macAddress: Mac Address
         macFormat: 'Invalid MAC address format.'
       vGpus:
+        warnings:
+          minimumAllocatable: It's highly recommended to select a vGpu with a number of allocatable devices greater then the number of nodes (Machine Count) to avoid "un-schedulable" errors after cluster updates.
```

```suggestion
          minimumAllocatable: It's highly recommended to select a vGPU with a number of allocatable devices greater then the number of nodes (Machine Count) to avoid "un-schedulable" errors after cluster updates.
```

To be consistent with the doc.